testing/matrix/linter/rules/rules.go

```diff
@@ -30,6 +33,8 @@ import (
 	"github.com/deckhouse/deckhouse/testing/matrix/linter/utils"
 )
 
+const defaultRegistry = "registry.deckhouse.io/deckhouse/fe"
```

Comment:
This one should not be hardcoded in tests. If some test string is required, it is better to use something abstract.

Reply:
ok, thanks, i change defaultRegistry to "registry.example.com/deckhouse"